installation/versions.env

```diff
@@ -0,0 +1,64 @@
+﻿global.docs.version=e848ee10
```

Comment:
committed by mistake?

Reply:
Nope, it's no purpose. It has to me kept here to overcome Jenkins inability to deploy pr builds to cluster. It has to be removed before merging to master, though.